src/app/js/public/NavBar.js

```diff
@@ -24,7 +14,15 @@ import Drawer from './Drawer';
 import Search from './search/Search';
 import GraphSummary from './graph/GraphSummary';
 import Favicon from './Favicon';
-import theme from '../theme';
+import jsonConfig from '../../../../config.json';
```

Comment:
Are you importing the WHOLE config, with its secrets and stuff on the frontend build?